Formula/ledger.rb

```diff
@@ -19,18 +19,11 @@ class Ledger < Formula
   depends_on "mpfr"
   depends_on "python@2"
 
-  # Upstream fix for boost 1.68, remove with next version
-  patch do
-    url "https://github.com/ledger/ledger/commit/c18a55f9.diff?full_index=1"
-    sha256 "2e652fc4b247b9c7cd482bd07aa57a66fc86597d7a564e6ccf93232700a6c8d8"
-  end
-
   def install
     ENV.cxx11
 
-    # Boost >= 1.67 Python components require a Python version suffix
-    inreplace "CMakeLists.txt", "set(BOOST_PYTHON python)",
-                                "set(BOOST_PYTHON python27)"
+    inreplace "doc/ledger3.texi", "Getting help, ,",
```

Comment:
Why?

Reply:
Until we have https://github.com/ledger/ledger/pull/1760, in the next version